scripts/package_antares_web.sh

```diff
@@ -66,9 +66,7 @@ rm $ANTARES_SOLVER_ZIPFILE_NAME
 
 if [[ "$OSTYPE" == "msys"* ]]; then
   echo "INFO: Moving executables in '$ANTARES_SOLVER_DIR'..."
-  mv "$ANTARES_SOLVER_FOLDER_NAME/bin/antares-$ANTARES_SOLVER_VERSION-solver.exe" "$ANTARES_SOLVER_DIR"
-  mv "$ANTARES_SOLVER_FOLDER_NAME/bin/sirius_solver.dll" "$ANTARES_SOLVER_DIR"
-  mv "$ANTARES_SOLVER_FOLDER_NAME/bin/zlib1.dll" "$ANTARES_SOLVER_DIR"
+  mv "$ANTARES_SOLVER_DIR/solver/Release/"* "$ANTARES_SOLVER_DIR"
```

Comment:
Note that this leaves an empty dir in the final package `AntaresWeb/antares_solver/solver/Release`, we'd better remove it.